### DISCUSSION TITLE
GitHub Pages Deployment Issue: 'Failed to load resource: the server responded with a status of 404'

### DISCUSSION THREAD

**Discussion by Yapper689** [community/community#131247] [Actions] [answered]
I'm trying to make a white canvas [website](https://github.com/Yapper689/Draw-App), it is successfully built and deployed, but then I have the error. I have built and preview the site on my PC and it works perfectly, so I genuinely have no clue as to why the site failed to work on github. I tried deploying to vercel as well, same issue. Any help would be very much appreciated.

**Comment by chrisjhchung** [comment]
Your base needs to match your repository name. I can see in your config file you have:

```
...
base: 'draw-app',
...
```

Try changing that to:

```
...
base: 'Draw-App',
...
```

**Reply by Yapper689** [reply, marked as answer]
Thanks so much for replying. I just tried all variants of the `base` value and they all produce the same error unfortunately. Then I tried using absolute urls, also failed. But then I moved the `scripts` file to `src` and now it works!

There must be some logic behind it, but I'm glad it's solved now.
